app/AppState/types/TransactionType.ts

```diff
@@ -3,7 +3,7 @@ import TxDetailType from './TxDetailType';
 export default interface TransactionType {
   type: 'Sent' | 'Received' | 'SendToSelf'; // like kind
   fee?: number;
-  confirmations: number;
+  confirmations: number | null;
```

Comment:
Why not represent `unconfimred` as 0 confirmations?   What does 0 confirmations mean?

Reply:
If the tx is not mined (unconfirmed + true) the App put this: `-` otherwise the App view the number of confirmations... because maybe `0 confirmations` can be confusing for the user... but maybe I'm wrong...

If I see:
- `-` I understand the tx is processing...
- 0 I understand the tx is processed, but wating for new blocks.

Reply:
I changed to number only.